### PR TITLE
feat(website): show text color token pairings

### PR DESCRIPTION
**Pull request by SiTaggart**
- feat(website): show text color accessible background pairings
- feat(website): story and swap the chevron icon

Description and useful links to discussions / issues / tickets

### Change One

Details of change...

### Change Two

Details of change...

## Checklist

- [ ] Work in a "Draft" branch whilst you are getting feedback to reduce Chromatic costs. Once you are ready for approval, convert the PR to "Ready to review"
- [ ] Ensure all `required` checks have passed
- [ ] Icon changes must have a product design review
- [ ] Website content changes must have a product design review
- [ ] At least two engineers have reviewed any code changes
- [ ] At least one engineering lead has reviewed any core package changes or repository infrastructure
- [ ] Website visual regression testing is run by adding `🕵🏻‍♀️ Run website visual regression` label

### CODE REVIEW

File: packages/paste-website/src/components/site-wrapper/sidebar/sidebar-disclosure/SidebarDisclosureButton.tsx [hunk 1]
```diff
@@ -64,14 +64,10 @@ const StyledButton = React.forwardRef<HTMLButtonElement, SidebarDisclosureButton
         as="span"
         alignItems="center"
         display="flex"
-        transform={props['aria-expanded'] ? 'rotate(0deg)' : 'rotate(-90deg)'}
+        transform={props['aria-expanded'] ? 'rotate(90deg)' : 'rotate(0deg)'}
```
Comment (TheSisb) [inline packages/paste-website/src/components/site-wrapper/sidebar/sidebar-disclosure/SidebarDisclosureButton.tsx:67]: question: why are we reversing this?

File: packages/paste-design-tokens/tokens/global/text-color.yml [hunk 2]
```diff
@@ -483,28 +483,28 @@ props:
...
-      - color-background-decorative-30
+      - color-background-decorative-30-weakest
   color-text-decorative-40:
     value: "{!palette-purple-60}"
     comment: Text color with no semantic meaning, used for decorative purposes only. Should generally be used with matching decorative background and/or border tokens.
     text_contrast_pairing:
       - color-background
       - color-background-body
-      - color-background-decorative-40
+      - color-background-decorative-40-weakest
```
Comment (SiTaggart) [inline packages/paste-design-tokens/tokens/global/text-color.yml:507]: incorrect annotations here. These tokens don't exist.

File: packages/paste-design-tokens/tokens/themes/dark/global/text-color.yml [hunk 3]
```diff
@@ -155,31 +155,15 @@ props:
   color-text-decorative-10:
     value: "{!palette-gray-20}"
     comment: Text color with no semantic meaning, used for decorative purposes only. Should generally be used with matching decorative background and/or border tokens.
-    text_contrast_pairing:
```
Comment (SiTaggart) [inline packages/paste-design-tokens/tokens/themes/dark/global/text-color.yml:158]: No need to declare the pairings at the theme level. The pairings are set at the system level, so every theme should meet the same token pair requirements as they are used in the same places across the theme.

File: packages/paste-design-tokens/tokens/themes/evergreen/aliases/box-shadow.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 imports:
   - ../../../aliases/box-shadow.yml
+  - ../../../aliases/offset.yml
   - ./color-palette.yml
   - ./color.yml
-  - ./offset.yml
```
Comment (SiTaggart) [inline packages/paste-design-tokens/tokens/themes/evergreen/aliases/box-shadow.yml:5]: copy pasta from the other theme that didn't need the empty file

File: packages/paste-design-tokens/tokens/themes/twilio/global/box-shadow.yml [hunk 5]
```diff
@@ -3,7 +3,6 @@ global:
   category: box-shadow
 imports:
   - ../../../global/box-shadow.yml
-  - ../aliases/box-shadow.yml
```
Comment (SiTaggart) [inline packages/paste-design-tokens/tokens/themes/twilio/global/box-shadow.yml:6]: redundant file

File: packages/paste-design-tokens/tokens/themes/twilio/global/text-color.yml [hunk 6]
```diff
@@ -49,10 +49,6 @@ props:
   color-text-decorative-30:
     value: "{!palette-green-80}"
     comment: Text color with no semantic meaning, used for decorative purposes only. Should generally be used with matching decorative background and/or border tokens.
-    text_contrast_pairing:
```
Comment (SiTaggart) [inline packages/paste-design-tokens/tokens/themes/twilio/global/text-color.yml:52]: More copy pasta

**Review by TheSisb — APPROVED** [review]
I'm seeing a strange issue where opening the popover near the bottom of the screen overflows the viewport and not the page's scroll area, but that's not a blocker for this. A ticket to investigate would be appreciated, however

**Review by serifluous — DISMISSED** [review]

**Review by libertycarlin — DISMISSED** [review]
there you goooooo

File: packages/paste-website/stories/TokenCard.stories.tsx [hunk 7]
```diff
@@ -91,6 +91,7 @@ const Template: StoryFn<typeof TokenCard> = ({name, category, useCamelCase}) =>
         textColorInverse={textColorInverse}
         borderColor={borderColor}
         useCamelCase={useCamelCase}
+        text_contrast_pairing={text_contrast_pairing}
```
Comment (shleewhite) [inline packages/paste-website/stories/TokenCard.stories.tsx:94]: Question: why is this prop snake case? Could it be like the others?

**Review by shleewhite — APPROVED** [review]

File: packages/paste-website/stories/TokenCard.stories.tsx (hunk 7 above)
Comment (SiTaggart) [reply packages/paste-website/stories/TokenCard.stories.tsx:94]: It totally could, essentially all this stuff is typed from the Tokens Object itself, and that's how it's named on the object, which is unfortunate. it's just lazy typing  to be able to pass the object around without too much hassle and map the props to the object keys

**Review by nkrantz — APPROVED** [review]

**Pull request merged**